server/server_test.go

```diff
@@ -591,6 +592,67 @@ func (cli *testServerClient) runTestLoadDataAutoRandomWithSpecialTerm(t *testing
 	})
 }
 
+func (cli *testServerClient) runTestLoadDataForGeneratedColumns(t *testing.T) {
```

Comment:
please move those test to `executer/loaddatatest`

Reply:
> please move those test to `executer/loaddatatest`

executor/loaddatatest does not support this kind of test.
Because the load data protocol is not the normal `handleQuery`, this test can't be done with `testkit.NewTestkit()`
In server package, the testkit is much like a integration test which handles the complete protocol layer.

@D3Hunter